pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -51,7 +54,8 @@ func NewPeriodicBackupController(
 	operatorImagePullSpec string,
 	accessor featuregates.FeatureGateAccess,
 	backupVarGetter backuphelpers.BackupVar,
-	backupsInformer factory.Informer) factory.Controller {
+	backupsInformer factory.Informer,
+	etcdPodsInformer factory.Informer) factory.Controller {
 
 	c := &PeriodicBackupController{
 		operatorClient:        operatorClient,
```

Comment:
```suggestion
		operatorClient:        operatorClient,
		podLister: kubeInformers.InformersFor(operatorclient.TargetNamespace).Core().V1().Pods().Lister(),
```

Reply:
This is exactly how i did it first run 😁😁

I have it in stash